src/frontend/components/Chat/index.tsx

```diff
@@ -27,11 +27,8 @@ export const Chat = ({ video: baseVideo, standalone }: ChatProps) => {
   }
 
   return (
-    <Box
-      align="start"
-      direction="row"
-      id="converse-container"
-      {...conditionalProps}
-    />
+    <Box {...conditionalProps}>
+      {!!standalone ? <div>Instructor Chat</div> : <div>Student Chat</div>}
+    </Box>
```

Comment:
nop, you should not update this component as long as your chat can not be fully used in production

Reply:
I changed my PR to not have any breaking changes. 
I rollbacked to 7.0.6 and changed my use of converse to be able to still display converse UI chat, along with my new features.